docs/options.md

```diff
@@ -21,11 +21,40 @@ or `axios` section in `nuxt.config.js`:
 }
 ```
 
+### Runtime Config
+
+The use of [runtime config](https://nuxtjs.org/guide/runtime-config) is mandatory in case of using environment variables in production, otherwise, the values will be hard coded during build and won't change.
+
```

Comment:
Would be neat mentioning what exact variables are supported (`browserBaseURL` and `baseURL`)